src/main/java/SpringJPA/Model/User.java

```diff
@@ -51,6 +55,21 @@ public User(String user, String pass, String mail, UserType role, long apiCallCo
         this.role = role;
         this.apiCallCount = apiCallCount;
         this.apiCallLimit = apiCallLimit;
+        this.startTime = System.currentTimeMillis();
+    }
```

Comment:
can you make this functionality into a public function

Reply:
Yes, I will do that real quick